src/lib/components/legend/Legend.svelte

```diff
@@ -42,167 +33,96 @@
 		{ value: 'fundingSource', label: 'Funding' }
 	];
 
-	function handleModeChange(event: Event) {
-		const target = event.target as HTMLInputElement;
-		visualState.update(state => ({ 
-			...state, 
-			colorMode: target.value as typeof state.colorMode,
+	function handleModeChange(event: Event & { currentTarget: HTMLInputElement }) {
```

Comment:
This is the slightly more conventional way to type event handlers in Svelte now, rather than doing a cast on `event.target`.